lib/middleware/lambdas.js

```diff
@@ -20,4 +20,5 @@ module.exports = function lambdas(req, res) {
       };
     }
   });
+  next();
 };
```

Comment:
this is typical - we don't always need options, we only include options if the middleware needs options - have a look at the middleware in wizard